_includes/tutorials/iot-loss-tracking/confluent/code/tutorial-steps/dev/process.sql

```diff
@@ -0,0 +1,28 @@
+SET 'auto.offset.reset' = 'earliest';
+
+-- Create a stream for raw telemetry data
+CREATE STREAM iot_telemetry (
+  device_id INT,
+  ts BIGINT
+) WITH (
+  KAFKA_TOPIC = 'iot_telemetry',
+  PARTITIONS = 1,
+  VALUE_FORMAT = 'AVRO',
```

Comment:
@davetroiano Unfortunately the recipe cannot use AVRO, must use JSON (because in-product tutorial does not enable CCSR)

Reply:
changed to JSON and removed SR from test